package.json

```diff
@@ -1,8 +1,14 @@
 {
-  "name": "ga-wdi-boston.js-boggle-challenge",
+  "name": "ga-wdi-boston.node-template",
   "version": "0.0.1",
   "private": true,
-  "license": "MIT",
+  "scripts":{
+    "start": "node bin/example.js"
```

Comment:
This is incorrect. In this case, it should be `"node bin/challenge.js"`. When you update from template, you may need to leave this block out if there are no scripts.